packages/styles/tag.css

```diff
@@ -15,18 +18,23 @@
 .Tag {
   color: var(--tag-text-color);
   background-color: var(--tag-background-color);
-  font-size: var(--text-size-smaller);
```

Comment:
I apologize, I messed up this requirement. The font size should be set to `--text-size-body-small` scoped to variable specific to tag.